### ISSUE TITLE
Aria label is missing for scrollbars - Accessibility

### ISSUE THREAD

**Issue by pavandixit93** [mdbootstrap/perfect-scrollbar#955]
Make sure these boxes are checked before submitting your issue -- thank you!
Hi Team,

Aria label is missing for horizontal and vertical scrollbars. During accessibility testing using screen readers like JAWS/NVDA we found that scrollbars are getting focused but the screen readers announces nothing as `aria-label` is not added on the elements.

Source code: 
https://github.com/mdbootstrap/perfect-scrollbar/blob/87ce05ef007ad11194d47160374fcccd96230a4d/src/index.js#L87
Here,  `this.scrollbarX.setAttribute('tabindex', 0);`,  tabindex is set alongwith tabindex, `aria-label` should be set. like,
this.scrollbarX.setAttribute('aria-label', 'Horizonatal scrollbar');

https://github.com/mdbootstrap/perfect-scrollbar/blob/87ce05ef007ad11194d47160374fcccd96230a4d/src/index.js#L115
Here, `this.scrollbarY.setAttribute('tabindex', 0);`,  tabindex is set alongwith tabindex, `aria-label` should be set. like,
this.scrollbarY.setAttribute('aria-label', 'Vertical scrollbar');
So that Sreen readers will annunce, `Horizonatal scrollbar` when horizonatal scrollbar is focused i.e. `.ps__thumb-x` and will announce `Vertical scrollbar` when vertical scrollbar is focused i.e. `.ps__thumb-y`

![image](https://user-images.githubusercontent.com/21095558/86559225-186d8480-bf79-11ea-8cb3-6ad944809f63.png)

Thank you....

- [ ] Check [FAQ](https://github.com/mdbootstrap/perfect-scrollbar/wiki/FAQ) and [Caveats](https://github.com/mdbootstrap/perfect-scrollbar/wiki/Caveats)
- [ ] Search if there's already one reported in Issues
- [ ] Prepare a JSFiddle reproducing the issue
  - [perfect-scrollbar JSFiddle](https://jsfiddle.net/utatti/dyvL31r6/)
- [ ] Provide a page or source code where the issue can be checked

**Comment by pavandixit93** [comment]
Hi Team, Any update on this please.

**Comment by pavandixit93** [comment]
Hi Team Any update on this?
